### ISSUE TITLE
The 480p checkpoint

### ISSUE THREAD

**Issue by Edwardmark** [PKU-YuanGroup/Open-Sora-Plan#357]
Hi，thanks for your great work. Where is the 480p checkpoint? The url is 404.

**Comment by LinB203** [comment]
Yes, we have open-sourced 29×720p and 93×480p models. Please note that the weights for 29×720p and 93×480p were trained on Panda70m and have not undergone final high-quality data fine-tuning, so they may produce watermarks.
Check the latest [resources](https://github.com/PKU-YuanGroup/Open-Sora-Plan?tab=readme-ov-file#-resource).
